docs/Tutorials/Create-Permissioned-Network.md

```diff
@@ -235,18 +226,18 @@ and [`--permissions-accounts-enabled`](../Reference/Pantheon-CLI-Syntax.md#permi
 * All domains can access the node using the HTTP JSON-RPC API using the [`--rpc-http-cors-origins`](../Reference/Pantheon-CLI-Syntax.md#rpc-http-cors-origins) option. 
 
 
-### 8. Start Node-2 
+### 7. Start Node-2 
 
 You need the [enode URL](../Configuring-Pantheon/Node-Keys.md#enode-url) for Node-1 to specify Node-1 as a bootnode. 
```

Comment:
After discussing this, it appears that you need enodes and addresses to create the permisionning files. Then you are in a chicken and egg situation, so you can't start the node to run the enode RPC method as it requires that the node is started with the file that already contains the enode in the whitelist. Wll ok, let's do this manually until we find a better way. But then perhaps we should explain more why we do it that way and no like in other tutorials.

Reply:
We could change the tutorial so we don't need to do this. I'm not sure if it will make it easier:
1. Start node1 and node2 with permissions enabled and no bootnodes
2. Grab the enodeURL of node1 and add it as a permitted node in node2 (using the API)
3. Grab the enodeURL of node2 and add it as a permitted node in node1 (using the API)
4. Add node2 as a peer of node1 using JSON-RPC

Reply:
I'm going to address these changes with the updates for PR #1118 (updates to permissioning command line options) so I'm merging this one.